src/Autofac.Pooling/Autofac.Pooling.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <LangVersion>latest</LangVersion>
+    <VersionPrefix>1.0.0</VersionPrefix>  
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AssemblyOriginatorKeyFile>../../Autofac.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>  
+    <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>    
+    <CodeAnalysisRuleSet>../../build/Analyzers.ruleset</CodeAnalysisRuleSet>
+    <GenerateAssemblyTitleAttribute>false</GenerateAssemblyTitleAttribute>
+    <GenerateNeutralResourcesLanguageAttribute>false</GenerateNeutralResourcesLanguageAttribute>
+    <GenerateAssemblyCopyrightAttribute>false</GenerateAssemblyCopyrightAttribute>
+    <GenerateAssemblyDescriptionAttribute>false</GenerateAssemblyDescriptionAttribute>
+  </PropertyGroup>
+
+  <!-- Nuget Options -->
+  <PropertyGroup>
+    <PackageTags>autofac;di;ioc;dependencyinjection;pooling</PackageTags>
+    <PackageReleaseNotes>Release notes are at https://github.com/autofac/Autofac.Pooling/releases</PackageReleaseNotes>
+    <PackageIconUrl>https://cloud.githubusercontent.com/assets/1156571/13684110/16b8f152-e6bf-11e5-84ae-22c66c6d351a.png</PackageIconUrl>
+    <PackageProjectUrl>https://autofac.org</PackageProjectUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <Features>Pooling</Features>
```

Comment:
What does `<Features>` do?

Reply:
Apparently nothing; I copied it from the Autofac csproj though! 😂

Reply:
OK, so, in early days of Roslyn analyzers, the `<Features>IOperation</Features>` was a feature flag needed to enable the analyzers. [Here's an issue that touches on it.](https://github.com/dotnet/roslyn/issues/9270) It's not needed anymore. It wasn't clear if there was something else going on here.